pudl/models/epacems.py

```diff
@@ -1,21 +1,28 @@
 """Database models for PUDL tables derived from EPA CEMS Data."""
 
-from sqlalchemy import Boolean, Integer, String, Float, Numeric, Date
-from sqlalchemy import Column, ForeignKey, ForeignKeyConstraint
+from sqlalchemy import Integer, String, Float, DateTime, Column
 import pudl.models.entities
 
 
 class HourlyEmissions(pudl.models.entities.PUDLBase):
     """Hourly emissions data by month as reported to EPA CEMS."""
+    # TODO: Add these constraints and indexes *after* loading all the data in.
+    # UniqueConstraint("orispl_code", "unitid", "op_date", "op_hour")
+    # indexes on: orispl_code, op_datetime, orispl_code, and the date part of
+    # op_datetime (each individually)
+    # Command to create an index for the date part of op_datetime
+    # CREATE INDEX op_date_idx (op_datetime::date)
 
-    __tablename__ = 'hourly_emissions_epacems'
+    __tablename__ = "hourly_emissions_epacems"
+    __table_args__ = {'prefixes': ['UNLOGGED']}
     id = Column(Integer, autoincrement=True, primary_key=True)  # surrogate key
-    state = Column(String)
-    facility_name = Column(String)
-    orispl_code = Column(Integer)
-    unitid = Column(String)
-    op_date = Column(String)
-    op_hour = Column(Integer)
+    state = Column(String, nullable=False)
+    facility_name = Column(String, nullable=False)
+    # TODO: Link to EIA plant ID
+    orispl_code = Column(Integer, nullable=False)
+    unitid = Column(String, nullable=False)
+    op_datetime = Column(DateTime, nullable=False)
+    # TODO: Make a view that multiplies op_time and gload_mw to get gload_mwh
```

Comment:
So op_time is the fraction of the hour that the plant was operating, and gload_mw is... the average power output over the course of the hour? We've generally been trying to expand the names of the columns in PUDL to make them more readable. So maybe we can use (e.g.) gross_load_mw and operating_datetime. Seems like operating_time really is an "interval" in postgresql speak -- an amount of time. Can we simplify the flag columns and make them ENUM types to enforce the small number of values they're allowed to have (whatever those are).  We've been trying to get away from the freeform strings where possible.